docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -942,3 +942,88 @@ spec:
   purpose: Informational
   tags: compliance, hunting
   contributors: nabilschear
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Firewall enabled, domain profile (Windows)
+  query: SELECT 1 FROM registry WHERE path LIKE 'HKEY_LOCAL_MACHINE\Software\Policies\Microsoft\WindowsFirewall\DomainProfile\EnableFirewall' AND CAST(data as integer) = 1;
+  description: "Checks for the existence of a Group Policy that enables the Domain Profile for Windows Defender Firewall."
+  resolution: "Contact your IT administrator to ensure your Windows system is receiving a Group Policy that enables the Domain profile for Windows Defender Firewall."
```

```suggestion
  resolution: "Contact your IT administrator to ensure your computer is receiving a Group Policy that enables the domain profile for Windows Firewall."
```

I suggest simplifying this a bit. Microsoft docs use "Windows Firewall": https://learn.microsoft.com/en-us/previous-versions/windows/desktop/ics/windows-firewall-profiles